rocminfo.cc

```diff
@@ -997,6 +1009,44 @@ AcquireAndDisplayAgentInfo(hsa_agent_t agent, void* data) {
   return HSA_STATUS_SUCCESS;
 }
 
+void CheckInitError(hsa_status_t err) {
+
+  printf("ROCm initialization failed: %d\n", err);
+
+  // Check kernel module for ROCk is loaded
+  FILE *fd = popen("lsmod | grep amdgpu", "r");
+  char buf[16];
+  if (fread (buf, 1, sizeof (buf), fd) <= 0) {
+    printf("ROCk module is NOT loaded, possibly no GPU devices\n");
+    return;
+  }
+
+  // Check if user belongs to group "video"
+  // @note: User who are not members of "video"
+  // group cannot access DRM services
+  int status = -1;
+  bool member = false;
+  char gr_name[] = "video";
+  struct group* grp = NULL;
+  do {
+    grp = getgrent();
+    if (grp == NULL) {
+      break;
+    }
+    status = memcmp(gr_name, grp->gr_name, sizeof(gr_name));
+    if (status == 0) {
+      member = true;
+      break;
+    }
+  } while (grp != NULL);
+  if (member == false) {
+    printf("User is not member of \"video\" group\n");
```

Comment:
Sys admins may change the name of the DRM access group.  This should probably read as a suggestion to check group status with video given as the name of the default DRM group.